include/d/actor/d_a_bpw.h

```diff
@@ -6,167 +6,159 @@
-    /* 0x4E8 */ u8 m4E8[0x4EC - 0x4E8];
-    /* 0x4EC */ u32 m4EC;
-    /* 0x4F0 */ u32 m4F0;
-    /* 0x4F4 */ u32 m4F4;
-    /* 0x4F8 */ u16 m4F8;
-    /* 0x4FA */ u16 m4FA;
-    /* 0x4FC */ u16 m4FC;
-    /* 0x4FE */ u8 m4FE[0x500 - 0x4FE];
-    /* 0x500 */ u32 m500;
-    /* 0x504 */ u32 m504;
-    /* 0x508 */ u8 m508[0x50C - 0x508];
+    /* 0x4E8 */ u32 m4E8;
+    /* 0x4EC */ LIGHT_INFLUENCE m4EC;
     /* 0x50C */ dPa_smokeEcallBack m50C;
     /* 0x52C */ dPa_followEcallBack m52C;
     /* 0x540 */ dPa_followEcallBack m540;
     /* 0x554 */ dPa_followEcallBack m554;
-    /* 0x568 */ dPa_followEcallBack m568;
-    /* 0x57C */ dPa_followEcallBack m57C;
+    /* 0x568 */ dPa_followEcallBack mFire1Dousa_Pa_followEcallBack;
+    /* 0x57C */ dPa_followEcallBack mFire1Dousa_Pa_followEcallBack2;
     /* 0x590 */ dPa_followEcallBack m590;
-    /* 0x5A4 */ dPa_followEcallBack m5A4;
-    /* 0x5B8 */ dPa_followEcallBack m5B8;
+    /* 0x5A4 */ dPa_followEcallBack mFireDousa2_Pa_followEcallBack;
+    /* 0x5B8 */ dPa_followEcallBack mFireDousa2_Pa_followEcallBack2;
     /* 0x5CC */ dPa_followEcallBack m5CC;
     /* 0x5E0 */ dPa_smokeEcallBack m5E0[4];
-    /* 0x660 */ f32 m660;
-    /* 0x664 */ f32 m664;
-    /* 0x668 */ f32 m668;
-    /* 0x66C */ u8 m66C[0x678 - 0x66C];
-    /* 0x678 */ u16 m678;
-    /* 0x67A */ s16 m67A;
-    /* 0x67C */ s16 m67C;
-    /* 0x67E */ u8 m67E[0x684 - 0x67E];
+    /* 0x660 */ cXyz mFire1DousaPos;
+    /* 0x66C */ cXyz m66C;
+    /* 0x678 */ csXyz mFire1DousaRot;
+    /* 0x67E */ s16 m67E;
+    /* 0x680 */ s16 m680;
+    /* 0x67E */ u8 m682[0x684 - 0x682];
     /* 0x684 */ s16 m684;
     /* 0x686 */ u8 m686[0x688 - 0x686];
-    /* 0x688 */ dBgS_AcchCir m688;
-    /* 0x6C8 */ dBgS_Acch m6C8;
-    /* 0x88C */ u8 m88C[0x8A4 - 0x88C];
-    /* 0x8A4 */ u32 m8A4;
-    /* 0x8A8 */ dCcD_GStts m8A8;
-    /* 0x8C8 */ dCcD_GObjInf m8C8;
-    /* 0x9C0 */ u8 m9C0[0x9D8 - 0x9C0];
-    /* 0x9D8 */ u32 m9D8;
-    /* 0x9DC */ u32 m9DC;
-    /* 0x9E0 */ u8 m9E0[0x9F0 - 0x9E0];
-    /* 0x9F0 */ u32 m9F0;
-    /* 0x9F4 */ dCcD_GObjInf m9F4;
-    /* 0xAEC */ u8 mAEC[0xB04 - 0xAEC];
-    /* 0xB04 */ u32 mB04;
-    /* 0xB08 */ u32 mB08;
-    /* 0xB0C */ u8 mB0C[0xB1C - 0xB0C];
-    /* 0xB1C */ u32 mB1C;
-    /* 0xB20 */ dCcD_GObjInf mB20;
-    /* 0xC18 */ u8 mC18[0xC30 - 0xC18];
-    /* 0xC30 */ u32 mC30;
-    /* 0xC34 */ u32 mC34;
-    /* 0xC38 */ u8 mC38[0xC48 - 0xC38];
-    /* 0xC48 */ u32 mC48;
-    /* 0xC4C */ dCcD_GObjInf mC4C;
-    /* 0xD44 */ u8 mD44[0xD5C - 0xD44];
-    /* 0xD5C */ u32 mD5C;
-    /* 0xD60 */ u32 mD60;
-    /* 0xD64 */ u8 mD64[0xD74 - 0xD64];
-    /* 0xD74 */ u32 mD74;
-    /* 0xD78 */ u8 mD78[0xD80 - 0xD78];
+    /* 0x688 */ dBgS_AcchCir mDamageBallSpeed;
+    /* 0x6C8 */ dBgS_ObjAcch mBodyBgS_Acch;
+    /* 0x88C */ dCcD_Stts mBodyCcD_Stts;
+    /* 0x8C8 */ dCcD_Sph mBodySph;
+    /* 0x9F4 */ dCcD_Sph mBodySph2;
+    /* 0xB20 */ dCcD_Sph mKanteraSph;
+    /* 0xC4C */ dCcD_Sph mDamageBallCcD_Sph;
```

Comment:
Why is this named "mDamageBallSpeed"? I don't think AcchCirs are related to speed.

Also, "mBodyBgS_Acch" has "mDamageBallSpeed" passed as its acch_cir, so the first part of the name should match between them, "body" and "damage ball" are too different and suggest that they are unrelated things rather than two parts of the same hitbox.

I would name them like this (based on the official `src` names):

```suggestion
    /* 0x688 */ dBgS_AcchCir mAcchCir;
    /* 0x6C8 */ dBgS_ObjAcch mAcch;
    /* 0x88C */ dCcD_Stts mStts;
    /* 0x8C8 */ dCcD_Sph mBodyCoSph;
    /* 0x9F4 */ dCcD_Sph mBodyAtSph;
    /* 0xB20 */ dCcD_Sph mKanteraCoSph;
    /* 0xC4C */ dCcD_Sph mDamageBallCoSph;
```